runtime/krest/src/lib.rs

```diff
@@ -955,6 +885,39 @@ impl peaq_pallet_storage::Config for Runtime {
 	type WeightInfo = peaq_pallet_storage::weights::SubstrateWeight<Runtime>;
 }
 
+// Zenlink-DEX Parameter definitions
+parameter_types! {
+	pub SelfParaId: u32 = ParachainInfo::parachain_id().into();
+
+	pub const ZenlinkDexPalletId: PalletId = PalletId(*b"zenlkpro");
+
+	pub ZenlinkRegistedParaChains: Vec<(MultiLocation, u128)> = vec![
+		// Krest local and live, 0.01 BNC
+		(MultiLocation::new(1, Junctions::X1(Junction::Parachain(2000))), 10_000_000_000),
```

Comment:
Just like the above comment, we have to discuss whether we should disable it in the Krest/Peaq network. However, I don't think other people transfer the token from chain 3000 to our chain.